widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/call/CallThemeMappers.kt

```diff
@@ -0,0 +1,48 @@
+package com.glia.widgets.view.unifiedui.theme.call
+
+import com.glia.widgets.view.unifiedui.config.call.BarButtonStates
+import com.glia.widgets.view.unifiedui.config.call.BarButtonStyle
+import com.glia.widgets.view.unifiedui.config.call.ButtonBar
+import com.glia.widgets.view.unifiedui.config.call.Call
+import com.glia.widgets.view.unifiedui.theme.base.updateFrom
+
+fun ThemeBarButtonStyle?.updateFrom(barButtonStyle: BarButtonStyle?): ThemeBarButtonStyle? =
+    barButtonStyle?.let {
+        ThemeBarButtonStyle(
+            background = this?.background.updateFrom(it.background),
```

Comment:
Just to clarify: this is a kind of overwriting the styles with the values from "unified UI", right?

Reply:
Yep, I also will add the same functional to overwrite parameters from current `UiTheme`